backend/danswer/db/persona.py

```diff
@@ -200,7 +202,16 @@ def get_personas(
     if not include_deleted:
         stmt = stmt.where(Persona.deleted.is_(False))
 
-    return db_session.scalars(stmt).all()
+    if joinedload_all:
```

Comment:
most of the time we access all of these anyways, so just load them all in one query